src/java.base/share/classes/java/security/cert/CertPathValidator.java

```diff
@@ -331,11 +331,8 @@ public final CertPathValidatorResult validate(CertPath certPath,
     public static final String getDefaultType() {
         @SuppressWarnings("removal")
         String cpvtype =
-            AccessController.doPrivileged(new PrivilegedAction<>() {
-                public String run() {
-                    return Security.getProperty(CPV_TYPE);
-                }
-            });
+            AccessController.doPrivileged((PrivilegedAction<String>) () ->
+                    Security.getProperty(CPV_TYPE));
```

Comment:
See too many, maybe we need a dedicated method in `sun.security.util.SecurityProperties`?